docs/user-docs/latest/plugins/installing-plugins-via-marketplace.md

```diff
@@ -4,37 +4,39 @@ description: Instructions to install new plugins in a Strapi application via the
 canonicalUrl: https://docs.strapi.io/user-docs/latest/plugins/installing-plugins-via-marketplace.html
 ---
 
-# Installing plugins via the Marketplace
+# Managing Marketplace plugins
 
-::: callout 🚧 The Marketplace is currently not available for v4. It will be back soon, stay tuned!
-In the meantime, plugins can be installed via the Command Line Interface (see [Developer Documentation](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-install)).
-:::
-
-The Marketplace is a section of the admin panel that lists all plugins that can be installed in a Strapi application. The Marketplace is accessible from ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_ in the main navigation of the admin panel.
-
-<!--
+The Marketplace is where users can find additional plugins to customize Strapi applications. The Marketplace is located in the admin panel, indicated by ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_. In the Marketplace, users can browse or search for plugins, link to detailed plugin descriptions and submit new plugins. This documentation describes how to navigate the Marketplace and how to install Marketplace plugins.
 
+::: caution
+ v3 and v4 plugins are not cross-compatible.
+:::
+<!-- Is it better to keep these two callouts separate or combine them into the caution callout? -->
 ::: note
-Plugins can also be installed via the Command Line Interface (see [Developer Documentation](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-install)).
+The Marketplace displays only v4 plugins. Both v3 and v4 plugins are discoverable in the [Strapi Market](https://market.strapi.io/).
 :::
 
-![Interface of the Marketplace](../assets/plugins/marketplace.png)
+![The Marketplace interface](../assets/plugins/marketplace-v4.png)
 
-The Marketplace displays each available plugin in a box, which contains:
-- the name of the plugin,
-- the description of the plugin,
-- indications on the current status of the plugin:
-   - "Compatible with your app": indicates that the plugin is not installed yet but can be installed on your Strapi application
-   - "Already installed": indicates that the plugin is already installed and available in your Strapi application
+Plugins are displayed on individual cards containing:
+
+- the plugin name, sometimes followed by either of the following badges:
+   - ![made by Strapi icon](../assets/icons/official-market.svg) to indicate the plugin is made by Strapi,
+   - ![verified by Strapi icon](../assets/icons/verified-marketplace.svg) to indicate the plugin was verified by Strapi.
+- the plugin description
+- a **Learn more** button for additional information, including detailed implementation instructions
+- a **Copy install command** button to copy the plugin installation command to the local clipboard
 
```

Comment:
Maybe it wasn't clear enough in my previous review but even though the possibility to submit plugins is not worth creating a dedicated section in the documentation, we still need to mention/explain the "Submit plugin" button.
It could be for example "In the top right corner of the Marketplace, a **Submit your plugin** button redirects to the Strapi Market from where it is possible to submit your own plugin" or something like that 🤓  Not in the bullet list above of course, but right after, in another paragraph.